# LightsControl.srv
int32[] lights
---
bool success
string message
